docs/KUBERNETES.md

```diff
@@ -68,6 +68,11 @@ kubectl apply -f k8s/agent-namespace.yaml
 kubectl create secret generic logdna-agent-key -n logdna-agent --from-literal=logdna-agent-key=<YOUR LOGDNA INGESTION KEY>
 kubectl apply -f k8s/agent-resources.yaml
 ```
+__Optionally:__ If you intend to use the K8's Lease option in the agent startup process, your cluster must be configured before starting the agent. To do this, run the following command in addition the commands listed above in step #2: 
+```console
+kubectl apply -f k8s/agent-startup-lease.yaml
```

Comment:
This command can run before applying the the agent-resources.yaml and after the create secret.

Reply:
I restructured, let me know if what I did makes sense.